components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImpl.java

```diff
@@ -733,7 +733,7 @@ public void updateAPI(String apiID, API substituteAPI) throws APIMgtDAOException
                 + "IS_RESPONSE_CACHED = ?, CACHE_TIMEOUT = ?, TECHNICAL_OWNER = ?, TECHNICAL_EMAIL = ?, " +
                 "BUSINESS_OWNER = ?, BUSINESS_EMAIL = ?, CORS_ENABLED = ?, CORS_ALLOW_ORIGINS = ?, " +
                 "CORS_ALLOW_CREDENTIALS = ?, CORS_ALLOW_HEADERS = ?, CORS_ALLOW_METHODS = ?, LAST_UPDATED_TIME = ?," +
-                "UPDATED_BY = ?, LC_WORKFLOW_STATUS=? WHERE UUID = ?";
+                "UPDATED_BY = ?, LC_WORKFLOW_STATUS=?, SECURITY_SCHEME=? WHERE UUID = ?";
```

Comment:
Please follow proper formatting here. notice the spaces before and after the = sign

Reply:
ack. will correct it